Maestro/subscriptions.json

```diff
@@ -296,7 +296,7 @@
       "actionArguments": {
         "vsoBuildParameters": {
           "ScriptFileName": "update-dependencies\\update-dependencies.ps1",
-          "Arguments": "-EnvVars 'GITHUB_USER=dotnet-bot','GITHUB_EMAIL=dotnet-bot@microsoft.com','GITHUB_PASSWORD=`$(`$Secrets[`'DotNetBotGitHubPassword`'])','CLI_BRANCH=rel/1.0.0'"
+          "Arguments": "-EnvVars 'GITHUB_USER=dotnet-bot','GITHUB_EMAIL=dotnet-bot@microsoft.com','GITHUB_PASSWORD=`$(`$Secrets[`'DotNetBotGitHubPassword`'])','CLI_BRANCH=rel/1.0.0','CLI_RELEASE_MONIKER=preview3'"
```

Comment:
I'd recommend using the array syntax for `Arguments` even though there's only one real arg to make it easier to look through, especially when columns are limited (like here).